test/api_integration/config.js

```diff
@@ -19,7 +19,10 @@ export default async function ({ readConfigFile }) {
     junit: {
       reportName: 'API Integration Tests',
     },
-    esTestCluster: commonConfig.get('esTestCluster'),
+    esTestCluster: {
+      ...functionalConfig.get('esTestCluster'),
+      serverArgs: ['xpack.security.enabled=false'],
```

Comment:
Is the plan to update the tests to work with security enabled?

Reply:
Yeah, that's the plan. I am working on putting together a meta issue to track/divvy all the necessary work to streamline the changes.